### ISSUE TITLE
Add support for latest nginx versions (1.26, 1.27)

### ISSUE THREAD

**Issue by tormi** [lando/php#131]
Slack thread for details: https://devwithlando.slack.com/archives/C2XBSHX8R/p1729140361030899

**Comment by pirog** [comment]
Agreed and support this ticket but FWIW this is not an error, just a warning, and in almost all cases shouldnt cause an issue.
